<launch>
  <arg name="args" default="-sgbm"/>	
  <param name="/use_sim_time" value="true"/>	
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find surface_detector)/config/config_file.rviz"/>
  <rosparam file="$(find surface_detector)/config/params.yaml" />			
  <node name="stereo_detector" type="stereo_detector.py" pkg="surface_detector" output="screen" args="$(arg args)">
	<param name="pkg_path" value="$(find surface_detector)"/>
  </node>
</launch>

